cou/steps/plan.py

```diff
@@ -237,6 +243,37 @@ def _verify_highest_release_achieved(analysis_result: Analysis) -> None:
         )
 
 
+def _verify_nova_cloud_controller_scheduler_default_filters(
+    args: CLIargs, analysis_result: Analysis
+) -> None:
+    """Verify scheduler_default_filters option is set correctly for each OpenStack release.
+
+    :param args: CLI arguments
+    :type args: CLIargs
+    :param analysis_result: Analysis result.
+    :type analysis_result: Analysis
+    """
+    if args.upgrade_group != CONTROL_PLANE:
+        return
```

Comment:
I think this verify function should always run, so that the user gets a picture of the whole cloud before an upgrade.

Reply:
That also make sense :+1:, My thought was that if they upgrade DATA_PLANE, they will not care about nova-cloud-controller/

Reply:
Yeah that is true - and if they're upgrading data-plane only, then that will only succeed if control-plane is already upgraded. Fine to leave as is. :+1:

Reply:
ah, I reverted

Reply:
I think both are fine, do you have a preference?

Reply:
revert my revert. on a second thought, if they only upgrade data plane, it does not make sense to ask them to update `scheduler_default_filters`

Reply:
Yep that makes sense. Should be fine considering the check. To be more accurate, we would need something like `if nova-cloud-controller would be upgraded and nova-cloud-controller.current_release == antelope`, but I'm not sure if COU would support that.